.projenrc.js

```diff
@@ -80,14 +83,21 @@ const project = new web.ReactTypeScriptProject({
 // synthesize project files before build
 // see https://github.com/projen/projen/issues/754
 const buildTask = project.tasks.tryFind("build");
+buildTask.spawn(project.packageTask);
+buildTask.prependSpawn(project.testTask);
```

Comment:
Am I missing something or did we not run tests as part of the build?

Reply:
I reckon there is a bug in `projen` whereby the `build` task (and I got a similar behavior on `compile`) appears to be reset by some action (was not able to quite pinpoint which though)... And then this causes some task dependencies to... get lost.